wpilibNewCommands/src/main/java/edu/wpi/first/wpilibj2/command/Command.java

```diff
@@ -466,4 +486,192 @@ enum InterruptionBehavior {
     /** This command continues, and the incoming command is not scheduled. */
     kCancelIncoming
   }
+
+  // Command Factories
+
+  /**
+   * Constructs a command that does nothing, finishing immediately.
+   *
+   * @return the command
+   */
+  static Command none() {
```

Comment:
Do we really want these functions visible on *every* Command class?  Wouldn't they be better as static functions in a separate Factory class?

Reply:
a separate `CommandFactory` class seems appropriate, yes

Reply:
let's not put `factory` in the name though

maybe just call it `Commands`

Reply:
about the only thing I really like about having these in `Command` is that `Command.run()` reads pretty nicely.  `Commands.run()` isn't bad either.